bitnami/tomcat/values.yaml

```
@@ -417,6 +420,26 @@ networkPolicy:
   ##    - {key: role, operator: In, values: [frontend]}
   ##
   explicitNamespacesSelector: {}
+
+## Service Account
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+##
+serviceAccount:
+  ## @param serviceAccount.create Enable creation of ServiceAccount for WordPress pod
```

Maybe?
```suggestion
  ## @param serviceAccount.create Enable creation of ServiceAccount for Tomcat pod
```

The README should be synced again